src/ety_main.erl

```diff
@@ -48,7 +48,7 @@ parse_args(Args) ->
          {only, $o, "only", string,
             "Only typecheck these functions (given as name/arity or just the name)"},
          {log_level,  $l,   "level",    string,
-            "Minimal log level (trace,debug,info,note,warn)"}
+            "Minimal log level (trace2,trace,debug,info,note,warn)"}
```

Comment:
Why the trace2 log level addition?

Reply:
because some trace log messages totally messed up the log. They now use trace2